project/AkkaBuild.scala

```diff
@@ -1428,11 +1458,17 @@ object AkkaBuild extends Build {
       "com.google.protobuf")
 
     def exports(packages: Seq[String] = Seq(), imports: Seq[String] = Nil) = osgiSettings ++ Seq(
-      OsgiKeys.importPackage := imports ++ defaultImports,
+      OsgiKeys.importPackage := imports,
+      OsgiKeys.importPackage <++= scalaVersion { v =>
```

Comment:
@jrudolph no worries, BND is smart and will add it to `Import-Package` header only when actually needed. I do this in this place, because it needs to be in front of `scala.*` to be effective.

@ktoso should I change the condition to `scalaBinaryVersion { v => if(v == "2.10") ...` or should it stay as is? I actually nicked that from cpsPlugin settings declaration at line 170